test/GPv2Settlement.test.ts

```diff
@@ -186,6 +186,62 @@ describe("GPv2Settlement", () => {
       expect(outTransfers.length).to.equal(tradeCount);
     });
 
+    it("should revert if the limit price is not respected", async () => {
+      const encoder = new SettlementEncoder(testDomain);
+      await encoder.signEncodeTrade(
+        {
+          ...partialOrder,
```

Comment:
I think it's the right tradeoff to keep the tests concise, but I had to actually check out the branch to understand what token our test order is buying/selling to verify the logic is correct. If all was created within the test it would tell one coherent story. That being said, I think it's an ok compromise as future maintainers will likely be in an editor and the info is just one click away.

Reply:
Maybe it would help if I added `sellToken` and `buyToken` variables?

Reply:
@fleupold Let me know if the change I just pushed makes things better.

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2F3oEjI5VtIhHvK37WYo%2Fgiphy.gif&token=nice" data-gitmeme-token="nice"><img src="https://media0.giphy.com/media/3oEjI5VtIhHvK37WYo/giphy.gif" title="Created by gitme.me with /nice"/></a>